src/titiler/xarray/pyproject.toml

```
@@ -30,24 +30,31 @@ classifiers = [
 dynamic = ["version"]
 dependencies = [
     "titiler.core==0.19.0.dev",
-    "cftime",
-    "h5netcdf",
     "xarray",
     "rioxarray",
-    "zarr",
     "fsspec",
-    "s3fs",
-    "aiohttp",
-    "pandas",
-    "httpx",
+    "zarr",
+    "h5netcdf",
+    "cftime",
 ]
 
 [project.optional-dependencies]
+s3 = [
+    "s3fs",
+]
+http = [
+    "aiohttp",
+]
+all = [
+    "s3fs",
+    "aiohttp",
+]
```

```suggestion
zarr = [
    "zarr",
]
hdf5 = [
    "h5netcdf",
]
all = [
    "s3fs",
    "aiohttp",
    "zarr",
    "hdf5",
]
```
Suggestion to align with above comment about optional dependencies

---

@maxrjones I'll take care of this either in the main PR or in another one later